README.md

```diff
@@ -49,44 +49,27 @@ Some examples:
 
 ### Trying In-development Neqo code in Gecko
 
-In a checked-out copy of Gecko source, set paths for the four Neqo crates to
-local versions in `netwerk/socket/neqo_glue/Cargo.toml`. For example, if Neqo
-was checked out to /home/alice/git/neqo, change:
-
-```
-neqo-http3 = { tag = "v0.1.7", git = "https://github.com/mozilla/neqo" }
-neqo-transport = { tag = "v0.1.7", git = "https://github.com/mozilla/neqo" }
-neqo-common = { tag = "v0.1.7", git = "https://github.com/mozilla/neqo" }
-```
-
-to
+In a checked-out copy of Gecko source, set `[patches.*]` values for the four
+Neqo crates to local versions in the root `Cargo.toml`. For example, if Neqo
+was checked out to `/home/alice/git/neqo`, add the following lines to the root
+`Cargo.toml`.
 
 ```
+[patch."https://github.com/mozilla/neqo"]
```

Comment:
The other way works, but `./mach vendor` will complain and this is more or less what is also done in [firefox-source-docs: Patching third-party crates](https://firefox-source-docs.mozilla.org/build/buildsystem/rust.html#patching-third-party-crates)